endpoints/openrtb2/amp_auction.go

```diff
@@ -60,6 +60,7 @@ func NewAmpEndpoint(
 	uuidGenerator uuidutil.UUIDGenerator,
 	ex exchange.Exchange,
 	validator openrtb_ext.BidderParamValidator,
```

Comment:
Do we still need `paramsValidator` with it now incorporated into the `requestValidator`?